src/node_options.cc

```diff
@@ -285,7 +285,7 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             kAllowedInEnvironment);
   AddAlias("--loader", "--experimental-loader");
   AddOption("--experimental-modules",
-            "experimental modules features",
+            "",
```

Comment:
For those wondering what this does. If an option does not have a description it does not print when someone does `node -h`